src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+immediate that Hegel indicates it by stating it is
+[not passing but has passed](#the-unity-of-being-and-nothing). As the categories
+are this unstable, effectively coming-to-be _is_ ceasing-to-be, and
+ceasing-to-be _is_ coming-to-be&mdash;emergence and cessation occur
+simultaneously within the same context. Therefore, `becoming`, if Hegel is
+right, is a double movement of rising and falling.
+
+### Sublation of Becoming
+
+#### Quiescent Result and Destruction
+
+> The equilibrium in which coming-to-be and ceasing-to-be are poised is in the
+> first place becoming itself. But this becoming equally collects itself in
+> _quiescent unity_. Being and nothing are in it only as vanishing; becoming
+> itself, however, is only by virtue of their being distinguished. Their vanish-
+> ing is therefore the vanishing of becoming, or the vanishing of the vanishing
+> itself. Becoming is a ceaseless unrest that collapses into a quiescent result
+> (Hegel 2010, 81/21.93).
+
+Hegel goes into more detail here on how coming-to-be and ceasing-to-be paralyze
+not only each other but the unity that constitutes them. Importantly, he points
+out how `becoming` depends on the _difference_ between `being` and `nothing`,
+but as it has been understood, this difference itself immediately dissolves
+since there is nothing determinate to support it. In other words, the difference
+itself emerges as it ceases. Given that the moments of `becoming` lose their
+difference, the dependencies of the movement are gone and with it `becoming`
+collapses into a quiescent result.
+
+Put differently, `becoming` depends on the vanishing of `being` and `nothing`,
+but this vanishing is itself unsustainable and thus vanishes, hence Hegel's
+phrase "the vanishing of the vanishing".
+
+> This can also be expressed thus: becoming is the vanishing of being into
+> nothing, and of nothing into being, and the vanishing of being and nothing in
+> general; but at the same time it rests on their being distinct. It therefore
+> contradicts itself in itself, because what it unites within itself is
+> self-opposed; but such a union destroys itself (Hegel 2010, 81/21.93-4).
+
+Hegel repeats the point, but sharpens the idea of an internal opposition within
+`becoming` itself. Its movement brings together contradictory impulses and is
+torn asunder by the unrelenting indeterminacy of `pure being` and `nothing`.
+
+`Becoming` is the attempt, one might say, to bring some coherence to the
+conflicting motions of the early categories by carefully noting what must be the
+case: `being` and `nothing` _are_ different, they _are_ the same and they vanish
+into one another. The tools by which `becoming` puts these divergent and
+incompatible ideas, such as moments, movement, coming-to-be and ceasing-to-be,
+are innovative and creative, but equally cannot contain the fury of negativity
+latent in within them. `Becoming` itself thus emerges and perishes.
```

Comment:
Good section on the sublation of Becoming. The last paragraph does a good job of really clarifying the issue.